commerce-payment-web/src/main/java/com/liferay/commerce/payment/web/internal/display/context/CommercePaymentMethodGroupRelsDisplayContext.java

```diff
@@ -223,6 +290,41 @@ public String getSelectedScreenNavigationCategoryKey() {
 				CATEGORY_KEY_COMMERCE_PAYMENT_METHOD_DETAILS);
 	}
 
+	public int getSubscriptionEntryCount(String engineKey)
+		throws PortalException {
+
+		ThemeDisplay themeDisplay = (ThemeDisplay)_renderRequest.getAttribute(
+			WebKeys.THEME_DISPLAY);
+
+		int activeSubscriptions = 0;
+
+		long groupId =
+			_commerceChannelLocalService.getCommerceChannelGroupIdBySiteGroupId(
+				themeDisplay.getSiteGroupId());
+
+		List<CommerceOrder> commerceOrders =
```

Comment:
Are we fetching all orders without pagination? Do we understand that this can be a very big number?

Reply:
All orders for the payment method they're disabling. I'll try to think of a better way for checking active subscriptions

Reply:
Dropping this whole method. Only needed for a custom message so not that important.